n_subscribers: 1000 # [-]

timer_rate: 100.0 # [Hz]
